pyg_lib/csrc/sampler/cpu/neighbor_kernel.cpp

```diff
@@ -24,13 +24,17 @@ template <typename node_t,
           typename temporal_t,
           bool replace,
           bool save_edges,
-          bool save_edge_ids>
+          bool save_edge_ids,
+          bool return_sampled_info>
 class NeighborSampler {
  public:
   NeighborSampler(const scalar_t* rowptr,
                   const scalar_t* col,
                   const std::string temporal_strategy)
-      : rowptr_(rowptr), col_(col), temporal_strategy_(temporal_strategy) {
+      : rowptr_(rowptr),
+        col_(col),
+        temporal_strategy_(temporal_strategy),
+        num_neighbors_counter(0) {
```

Comment:
can we rename this to `curr_hop` or something like this?

Reply:
Done